assignments/objects.js

```diff
@@ -57,21 +57,34 @@ let antonietta = {
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(mitzi["name"]);
 
 // Kennan's ID
+console.log(kennan["id"]);
 
 // Keven's email
+console.log(keven["email"]);
 
 // Gannie's name
+console.log(gannie["name"]);
 
 // Antonietta's Gender
+console.log(antonietta["gender"]);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+kennan.speak = function() {
+  return "Hello, my name is Kennan!";
+};
+console.log(kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+
+antonietta.multiplyNums = function(a,b) {
+  return a * b;
+}
+console.log(antonietta.multiplyNums(3,4));
 
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
```

Comment:
Great work on objects! Try at the stretch problem when you get a chance. You are doing great!